### DISCUSSION TITLE
V3 Documentation error

### DISCUSSION THREAD

**Discussion by DanielGamboa** [spatie/image#276] [General]
i found a small error in the documentation:

https://spatie.be/docs/image/v3/image-manipulations/adding-a-watermark

Watermark fit resize
To change the way the watermark is resized within the given boundaries you can use the $fit param. This param accepts any Fit enum value.

You can read more about resizing using the fit methods in the [resizing images]**(https://spatie.be/image/v3/image-manipulations/resizing-images)** part of the docs.

URL should read: https://spatie.be/docs/image/v3/image-manipulations/resizing-images 

If someone could pass it along that would be perfect.

**Comment by timvandijck** [comment]
Thank you for reporting, I updated the link!
